src/main/character/Character.java

```diff
@@ -191,7 +191,7 @@ public void animate(Graphics2D canvas) {
 
 	private void updatePosition() {
 
-		int value = this.jumping ? -2 : 1;
+		int value = this.jumping ? (-Constants.MOVING_FACTOR * 2) : Constants.MOVING_FACTOR;

```

Comment:
It would be better to multiplicate value times Constants.MOVING_FACTOR
```java
this.getPosition().setY(this.getPosition().getY() + value * Constants.MOVING_FACTOR);
```